Modules/CalcOffence-3_0.lua

```diff
@@ -1172,13 +1178,82 @@ function calcs.offence(env, actor, activeSkill)
+				for index, value in ipairs(actor.activeSkillList) do
+					if value.activeEffect.gemData.name == "Seismic Cry" then
+						-- recursively calculate the values for Seismic Cry
+						-- only actual Cooldown and WarcryCastTime are returned
+						local seismicCryData = calcs.offence(env, actor, actor.activeSkillList[index], true)
+
+						output.SeismicCryCooldown = seismicCryData.Cooldown
+						output.SeismicCryCastTime = seismicCryData.WarcryCastTime
+						break
+					end
+				end
+				output.SeismicDmgImpact = 0
+				for i = 1, numSeismicExerts do
+					output.SeismicDmgImpact = output.SeismicDmgImpact + (i * moreDmgAndAoEPerExert)
+				end
+				output.SeismicAvgDmg = output.SeismicDmgImpact / numSeismicExerts
+				-- calculate ratio of uptime versus downtime (including opportunity cost of casting the warcry)
+				output.SeismicUpTimeRatio = m_min((numSeismicExerts / output.Speed) / (output.SeismicCryCooldown + output.SeismicCryCastTime), 1.0)
+				exertedUptime = m_max(exertedUptime, output.SeismicUpTimeRatio)
+				output.SeismicHitEffect = 1 + (output.SeismicAvgDmg * output.SeismicUpTimeRatio)
+				skillModList:NewMod("AreaOfEffect", "MORE", m_floor(output.SeismicAvgDmg * output.SeismicUpTimeRatio * 100), "Avg Seismic Exert AoE")
+			end
+		end
+
+		-- Intimidating Cry Exerts Attacks
+		output.IntimidatingHitEffect = 1
+		if activeSkill.skillTypes[SkillType.Attack] then
+			local numIntimidatingExerts = env.modDB:Sum("BASE", nil, "NumIntimidatingExerts") or 0
+			if numIntimidatingExerts > 0 and not skillFlags.warcry then
+				for index, value in ipairs(actor.activeSkillList) do
+					if value.activeEffect.gemData.name == "Intimidating Cry" then
+						-- recursively calculate the values for Intimidating Cry
+						-- only actual Cooldown and WarcryCastTime are returned
+						local intimidatingCryData = calcs.offence(env, actor, actor.activeSkillList[index], true)
+
+						output.InitmidatingCryDuration = intimidatingCryData.Duration
+						output.IntimidatingCryCooldown = intimidatingCryData.Cooldown
+						output.IntimidatingCryCastTime = intimidatingCryData.WarcryCastTime
+						break
+					end
+				end
+				-- calculate ratio of uptime versus downtime
+				output.IntimidatingUpTimeRatio = m_min((numIntimidatingExerts / output.Speed) / (output.IntimidatingCryCooldown + output.IntimidatingCryCastTime), 1.0)
+				exertedUptime = m_max(exertedUptime, output.IntimidatingUpTimeRatio)
+				-- intimidating cry guarantees double damage for its attacks; therefore, its hit effect
+				-- is calculated as the improvement over the non-intimidated double damage chance
+				output.IntimidatingHitEffect = 1 + (1 - output.DoubleDamageChance / 100) * output.IntimidatingUpTimeRatio
+
+				-- overwhelm calculation
+				local buffUptime = m_min(output.InitmidatingCryDuration / output.IntimidatingCryCooldown, 1)
+				skillModList:NewMod("EnemyPhysicalDamageReduction", "BASE", -30 * buffUptime, "Intimidating Cry Buff")
```

Comment:
The Enemy Phys Reduction from Intimidating Cry shouldn't be a constant -30; it's based on the Power of nearby enemies. I implemented Warcry Power already, and it's set up as Multiplier:WarcryPower. The Warcry grants 5% Overwhelm per 5 Power, up to 30% Overwhelm at 30 Power. That Overwhelm value should then also be scaled by buff effect.